libindy/android.test.sh

```diff
@@ -40,10 +40,8 @@ build_test_artifacts(){
         # TODO empty for full testing SET_OF_TESTS=''
         SET_OF_TESTS='--lib --test interaction'
 
-        # build - separate step to see origin build output
-        # TODO move RUSTFLAGS to cargo config and do not duplicate it here
         RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -lc -lz -L${TOOLCHAIN_DIR}/${TRIPLET}/lib -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -lsodium -lzmq -lgnustl_shared" \
-                    cargo test ${BUILD_TYPE} --target=${TRIPLET} ${SET_OF_TESTS} --no-run
+            cargo build ${BUILD_TYPE} --target=${TRIPLET}
```

Comment:
it's important to keep 2 steps: the first one for full build of tests and the second one for parsing tests to running